vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSequencesController.java

```diff
@@ -54,12 +60,20 @@ public String sequence(Model model, @PathVariable("sequenceId") String sequenceI
         if (sequenceExist==null) {
             throw new SequenceNotFoundException(sequenceId);
         }
-        
+
         List<ISlide> slides = sequenceManager.getSequence(sequenceId).getSlides();
         if (slides.size() == 0) {
             throw new SlidesInSequenceNotFoundException();
         }
+
+        if(choiceId==null) {
+            choiceId=sequenceId;
+            branchingPointId = sequenceManager.getSequence(sequenceId).getSlides().get(0).getId();
```

Comment:
you should check if there are slides in the sequence first or you risk a null pointer or out of bounds exception

Reply:
This check is implemented one step above this block and exception is getting thrown, I will use the same list variable to get the first slide in the sequence.

Reply:
ah, right.